### PR TITLE
Sync UP4 plugin with the latest pipeline

### PR DESCRIPTION
**Pull request by osinstom**
This PR syncs UP4 plugin with the latest pipeline, but does not handle application filtering rules. UP4 termination tables are configured with "don't care" app_id.

### CODE REVIEW

**Comment by osinstom** [comment]
CC: @daniele-moro

**Review by daniele-moro — APPROVED** [review]

File: pfcpiface/p4rt_translator.go [hunk 1]
```diff
@@ -507,19 +510,28 @@ func (t *P4rtTranslator) buildUplinkTerminationsEntry(pdr pdr, shouldDrop bool,
 		return nil, err
 	}
 
+	// FIXME: replace app_id with a meaningful value once we implement the full support for app filtering
+	if err := t.withExactMatchField(entry, FieldApplicationID, uint8(0)); err != nil {
```
[the remark this answers is not in the document]
Comment (osinstom) [reply pfcpiface/p4rt_translator.go:514]: Does it make sense? It's a temporary change and I will replace that soon.

File: pfcpiface/p4rt_translator.go [hunk 2]
```diff
@@ -507,19 +510,28 @@ func (t *P4rtTranslator) buildUplinkTerminationsEntry(pdr pdr, shouldDrop bool,
...
+		return nil, err
+	}
+
 	var action *p4.Action
 	if shouldDrop {
 		action = &p4.Action{
 			ActionId: t.actionID(ActUplinkTermDrop),
 		}
-	} else {
+	} else if !shouldDrop && tc != 0 {
```
[the remark this answers is not in the document]
Comment (osinstom) [reply pfcpiface/p4rt_translator.go:523]: the FIXME/TODO for that is already in up4.go

File: pfcpiface/p4rt_translator.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (daniele-moro) [reply pfcpiface/p4rt_translator.go:514]: I believe that constant is used in #410 as the default behavior (here: https://github.com/omec-project/upf-epc/pull/410/files#diff-b2e5401707df74f20e19da386241442387f6ef81557463d84d849f7f9ebe6c45R692). Defining a constant helps as intrinsic documentation as well. Maybe just keep this comment valid for #410 and leave this as is.

**Review by ccascone — APPROVED** [review]

**Review by pudelkoM — APPROVED** [review]

File: pfcpiface/p4rt_translator.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (osinstom) [reply pfcpiface/p4rt_translator.go:514]: let's keep this comment valid for #410. This line will be changed in that PR.

**Pull request merged**